dev-requirements.txt

```diff
@@ -3,7 +3,7 @@ graphviz >= 0.8.3
 jinja2 >= 2.0, < 3.0
 nbformat >= 4.4.0, < 5.0
 pre-commit >= 1.12.0, < 2.0
-pytest >= 3.8, < 4.0
+pytest >= 4.3, < 4.4
```

Comment:
Would you mind pinning the ceiling to 5.0? Hopefully that will be a safe range without constraining any other requirements people may have.